components/expandable-section/src/expandable-section.ts

```diff
@@ -55,38 +53,31 @@ export class DtExpandableSectionHeader {}
   preserveWhitespaces: false,
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class DtExpandableSection implements CanDisable {
+export class DtExpandableSection implements CanDisable, OnChanges {
   /** Whether the expandable section is expanded. */
   @Input()
   get expanded(): boolean {
     return this._panel.expanded;
   }
   set expanded(value: boolean) {
-    this._panel.expanded = value;
+    this._panel.expanded = coerceBooleanProperty(value);
     this._changeDetectorRef.markForCheck();
   }
 
-  /** Whether the expandable section is disabled. */
+  /** Whether the expandable section is expanded. */
```

Comment:
Why change it to "expanded" when the input says "disabled"?

Reply:
The git diff is odd :D it is a copy pasta 🍝 error because I copied the input with doc from the expanded input some lines above :D